contrib/babelfishpg_tsql/expected/test/babel_function.out

```diff
@@ -1567,11 +1567,9 @@ select dateadd(millisecond, 123, '20060830'::datetime);
 (1 row)
 
 select dateadd(microsecond, 123456, '20060830'::datetime);
-           dateadd            
-------------------------------
- Wed Aug 30 00:00:00.123 2006
-(1 row)
-
+ERROR:  The datepart microsecond is not supported by date function dateadd for data type time.
```

Comment:
The error message doesn't follow https://www.postgresql.org/docs/current/error-style-guide.html

`microsecond datepart not supported for the "datetime" data type`